src/main/java/ch/njol/skript/variables/VariablesStorage.java

```diff
@@ -69,7 +70,12 @@ public abstract class VariablesStorage implements Closeable {
 	/**
 	 * The name of the database, i.e. this storage.
 	 */
-	protected final String databaseName;
+	protected String databaseName;
+
+	/**
+	 * The type of the database, i.e. this storage.
+	 */
+	protected final String databaseType;
```

Comment:
Will need to change extending classes to be using the method instead of the field.

```suggestion
	private String databaseName;

	/**
	 * The type of the database, i.e. this storage.
	 */
	private final String databaseType;
```